setup.py

```diff
@@ -42,19 +42,19 @@ def get_version(version_file):
 
 
 setup(
-    name="example",
+    name="pe_source",
     # Versions should comply with PEP440
-    version=get_version("src/example/_version.py"),
-    description="Example Python library",
+    version=get_version("src/pe_source/_version.py"),
+    description="Posture and Exposure Source Library",
     long_description=readme(),
     long_description_content_type="text/markdown",
     # Landing page for CISA's cybersecurity mission
     url="https://www.cisa.gov/cybersecurity",
     # Additional URLs for this project per
     # https://packaging.python.org/guides/distributing-packages-using-setuptools/#project-urls
     project_urls={
-        "Source": "https://github.com/cisagov/pe-source",
-        "Tracker": "https://github.com/cisagov/pe-source/issues",
+        "Source": "https://github.com/cisagov/pe-reports",
+        "Tracker": "https://github.com/cisagov/pe-reports/issues",
```

Comment:
I'm not sure why these were changed:
```suggestion
        "Source": "https://github.com/cisagov/pe-source",
        "Tracker": "https://github.com/cisagov/pe-source/issues",
```

Reply:
I agree.  This is very confusing, and these files are definitely in the cisagov/pe-source repo.